source/guides/guides/cross-browser.md

```diff
@@ -0,0 +1,232 @@
+
+{% note info %}
+**Note:** The `name` under each `cypress/run` job which will be shown in the Circle CI workflow UI to distinguish the jobs.
+{% endnote %}
+
+```yaml
+version: 2.1
+orbs:
+  cypress: cypress-io/cypress@1
+workflows:
+  build:
+    jobs:
+      - cypress/install
+      - cypress/run:
+          name: Chrome
+          requires:
+            - cypress/install
+          executor: cypress/browsers-chrome73-ff68
+          start: npm start
+          wait-on: http://localhost:3000
+          record: true
+          group: chrome
+          browser: chrome
+      - cypress/run:
+          name: Firefox
+          requires:
+            - cypress/install
+          executor: cypress/browsers-chrome73-ff68
+          start: npm start
+          wait-on: http://localhost:3000
+          record: true
+          group: firefox-critical-path
+          browser: firefox
+          spec: "cypress/integration/signup.spec.js,cypress/integration/login.spec.js"
+```
+
+## Parallelize Per Browser (Group) Basis
+
+Execution of test files can be parallelized on a per {% url "group" parallelization.html#grouping-test-runs %} basis, where test files can be grouped by the browser under test. This versatility enables the ability to allocate the desired amount of CI resources towards a browser to either improve test duration or to minimize CI costs. 
+
+**You do not have to run all browsers at the same parallelization level.** In the example below, the Chrome dedicated `cypress/run` job runs *all* tests in parallel, across **4 machines**, against Chrome and reports results to the [Cypress Dashboard](https://on.cypress.io/dashboard) under the group name `chrome`. The Firefox dedicated `cypress/run` job runs a *subset* of tests in parallel, across **2 machines**, defined by the `spec` parameter, against the Firefox browser and reports results to the [Cypress Dashboard](https://on.cypress.io/dashboard) under the group named `firefox`.
+
+```yaml
+version: 2.1
+orbs:
+  cypress: cypress-io/cypress@1
+workflows:
+  build:
+    jobs:
+      - cypress/install
+      - cypress/run:
+          name: Chrome
+          requires:
+            - cypress/install
+          executor: cypress/browsers-chrome73-ff68
+          record: true
+          start: npm start
+          wait-on: http://localhost:3000
+          parallel: true
+          parallelism: 4
+          group: chrome
+          browser: chrome
+      - cypress/run:
+          name: Firefox
+          requires:
+            - cypress/install
+          executor: cypress/browsers-chrome73-ff68
+          record: true
+          start: npm start
+          wait-on: http://localhost:3000
+          parallel: true
+          parallelism: 2
+          group: firefox
+          browser: firefox
+          spec: "cypress/integration/app.spec.js,cypress/integration/login.spec.js,cypress/integration/about.spec.js"
+```
+
+## Running Specific Tests by Browser
```

Comment:
```suggestion
## Ignore some tests
```

Reply:
It's not just about ignoring tests. We also cover prescribe the inverse.